src/components/Navigation.jsx

```diff
@@ -48,31 +52,32 @@ class Navigation extends Component {
           type: "SET_PROJECT_NAME",
           payload: res.data.projectName,
         })
+        // set project ID
+        dispatch({
+          type: "SET_PROJECT_ID",
+          payload: value,
+        })
         this.props.history.push(`/project/${value}`);
         // window.location.href = `/project/${value}`;
       })
       .catch((error) => {
         console.log({ error });
         if (error.response.status === 404) {
-          // project Not found, pop an alert
-          // window.alert("Project ID not found!")
+          this.setState({projectExists: false})
           dispatch({
             type: "SET_PROJECT_EXISTS",
             payload: false,
           })
         }
```

Comment:
Do we still need this if statement here?

Reply:
good point, I think we can remove it